src/index.js

```diff
@@ -23,8 +23,8 @@ const hashOf = name => {
 		.slice(0, 10);
 }
 
-const nodeLoader = require(__dirname + "/loaders/node-loader.js");
-const relocateLoader = require(__dirname + "/loaders/relocate-loader.js");
+const nodeLoader = eval('require(__dirname + "/loaders/node-loader.js")');
+const relocateLoader = eval('require(__dirname + "/loaders/relocate-loader.js")');
```

Comment:
Why not `require("./loaders/node-loader.js")` and define it as `external`

Reply:
I just tried this and it doesn't work (get `Cannot read "assets" of undefined` indicating there may be separate instances in play, or something else).